code/modules/multiz/structures.dm

```diff
@@ -215,7 +215,7 @@
 			above.ChangeTurf(/turf/simulated/open)
 	. = ..()
 
-/obj/structure/stairs/Uncross(atom/movable/A)
+/obj/structure/stairs/Exit(atom/movable/A)
 	if(A.dir == dir && upperStep(A.loc))
 		// This is hackish but whatever.
```

Comment:
is this still hackish?

Reply:
Yeah kinda, I _think_ the new Exit proc means we could drop the if statement & upperStep proc